hotsos/plugin_extensions/openstack/agent/events.py

```diff
@@ -212,6 +211,30 @@ def __call__(self, event):
             if ret:
                 return {event.name: ret}, agent
 
+            return None
+
+        # All other events should be covered by the following
+        if event.name == 'ovn-resource-revision-bump':
```

Comment:
Should this and the two above be 'elif' ?

Reply:
Think that will trigger https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-return.html

Reply:
right, if you do that it will trigger pylint errors